gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Can you make the gs:// link in code font?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I don't know a way to combine a link with code format in markdown, do you?</p>

Reply:
You can do this:

```
[`gs://link`](LiveURL)
```

[`gs://link`](LiveURL)

Reply:
<p>Awesome - thank you!</p>